src/engine/enginebuffer.cpp

```diff
@@ -437,17 +440,11 @@ void EngineBuffer::requestSyncMode(SyncMode mode) {
 }
 
 void EngineBuffer::readToCrossfadeBuffer(const int iBufferSize) {
-    // During startup, scaler is null.
-    if (m_pScale == NULL) {
-        return;
-    }
     CSAMPLE* fadeout = m_pScale->getScaled(iBufferSize);
     SampleUtil::copy(m_pCrossfadeBuffer, fadeout, iBufferSize);
 
     // Restore the original position that was lost due to getScaled() above
     m_pReadAheadManager->notifySeek(m_filepos_play);
-    // Clear the current scale information
-    m_pScale->clear();
```

Comment:
this will cause the linear scalar to interpolate from the last sample of the old location.  This could cause pops.  We do crossfading so this may not be noticable, but removing this line is incorrect.  What are you fixing?